packages/contracts/src/contract-defs.ts

```diff
@@ -14,11 +15,13 @@ export const getContractDefinition = (name: string): any => {
 }
 
 export const getContractInterface = (name: string): ethers.utils.Interface => {
+  //returns the interface for a given contract by name.
   const definition = getContractDefinition(name)
   return new ethers.utils.Interface(definition.abi)
 }
 
 export const getContractFactory = (
+  //returns the interface,bytecode and signer of a given contract by name.
```

Comment:
These are non standard jsdoc comments, usually they go above the function definition with a particular format

Reply:
Thank you so much! My bad. I would correct my PR.

Reply:
New comments look great! Thank you :-)

Reply:
Got the new comments..would work on those! Thanks!

Reply:
Done. changeset added and rebase done.